kernelcare/README.md

```diff
@@ -0,0 +1,54 @@
+# Agent Check: kernelcare
+
+## Overview
+
+This check monitors [kernelcare][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+To install the kernelcare check on your host:
+
+1. Install the [developer toolkit](https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit) on any machine.
+2. Run `ddev release build kernelcare` to build the package.
+3. [Download the Datadog Agent](https://app.datadoghq.com/account/settings#agent).
+4. Upload the build artifact to any host with an Agent and run `datadog-agent integration install -w path/to/kernelcare/dist/<ARTIFACT_NAME>.whl`.
+
+### Configuration
+
+1. Edit the `kernelcare.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your kernelcare performance data. See the [sample kernelcare.d/conf.yaml][3] for all available configuration options.
+
+2. [Restart the Agent][4].
+
+### Validation
+
+[Run the Agent's status subcommand][5] and look for `kernelcare` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][6] for a list of metrics provided by this check.
+
+### Service Checks
+
+kernelcare does not include any service checks.
```

Comment:
It is recommended to have at least a can_connect service check that will be ok if everything is ok and critical if any of the requests fail

Reply:
added can_connect service check